examples/experimental/external_assets/airflow_example.py

```diff
@@ -0,0 +1,22 @@
+from airflow import DAG
+from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import (
+    KubernetesPodOperator,
+)
+from pendulum import datetime
+
+default_args = {
+    "start_date": datetime(2023, 10, 1),
+}
+
+with DAG(dag_id="example_kubernetes_pod", schedule="@daily", default_args=default_args) as dag:
+    KubernetesPodOperator(
+        image="asset-materialization-image:latest",
+        cmds=["python", "create_asset.py", "{{ ds }}"],
```

Comment:
`cmds=["python", "create_asset.py", "--execution-date", "{{ ds }}"]`

Reply:
Mean to give more context here. I adding `--execution-date` serves as documentation as to what `ds`, which might be a little obscure to some